numpy/core/src/common/npy_ctypes.h

```diff
@@ -32,6 +32,7 @@ npy_ctypes_check(PyTypeObject *obj)
     }
 
     ret = PyObject_IsTrue(ret_obj);
+    Py_DECREF(ret_obj);
```

Comment:
Nice find!

Reply:
Have a whole bunch more, I think refcount testing might be a big soon (except f2py probably), that would be awesome...